src/app/components/cards/TransferHistoryRow.scss

```diff
@@ -0,0 +1,18 @@
+.Memo--badActor {
+  .bad-actor-caution {
+    color: $color-red;
+    font-size: 75%;
+    font-weight: bold;
+    text-transform: uppercase;
```

Comment:
Can we please include some letter-spacing to the 'CAUTION'
e.g. `letter-spacing: 2px`